src/abstract/interfaces/IThresholdGovernor.sol

```diff
@@ -52,9 +60,32 @@ interface IThresholdGovernor is IBatchGovernor {
             uint256 noVotes,
             uint256 yesVotes,
             address proposer,
-            uint16 thresholdRatio
+            uint256 quorum,
```

Comment:
how about keeping it `uint16 thresholdRatio, uint256 quorum` vs `uint256 quorum, uint16 quorumNumerator` ?